snapshot/pkg/apis/crd/v1/types.go

```diff
@@ -298,6 +300,16 @@ func GetSupportedVolumeFromPVSpec(spec *core_v1.PersistentVolumeSpec) string {
 	if spec.Glusterfs != nil {
 		return "glusterfs"
 	}
+	if spec.CSI != nil {
+		switch spec.CSI.Driver {
+		case "pxd.portworx.com":
```

Comment:
Can you specify these as constants, so that stork can use the same instead of redefining it there

Reply:
👍